modules/upnp/doc_classes/UPNP.xml

```diff
@@ -16,6 +16,32 @@
 		[codeblock]
 		upnp.delete_port_mapping(port)
 		[/codeblock]
+		[b]Note:[/b] UPnP discovery blocks the main thread. To perform non-blocking discovery, use [Thread]s like this:
```

Comment:
> UPnP discovery blocks the main thread.

"UPnP discovery blocks the current thread. To perform discovery without blocking the main thread use [Thread]s like this:"